docs/api-docs/webhooks/about-webhooks.mdx

```diff
@@ -79,6 +79,18 @@ To acknowledge a callback has been received without issue, the destination serve
 
 Need to set up a quick webhook destination URL for testing? See [Tools for Debugging and Testing Webhooks](#tools).
 
+### Response requirements
+
+BigCommerce doesn't recommend sending bodies or headers in response to webhooks. 
+
+BigCommerce has the following webhook response requirements:
+
+**Limitations**
```

Comment:
```suggestion
If you must send response headers, they have to conform to this **Limitations** in order for the webkook delivery to be successfully acknowledged.
```

Reply:
@zvuki I addressed your feedback. 

If you send response headers, they must conform to the following limitations for the webhook delivery to be successfully acknowledged.